src/assets/sass/website/shared/footer.scss

```diff
@@ -2,15 +2,206 @@
 @import "../variables.scss";
 @import "../color-palette.scss";
 
-.footer-wrapper {
-  padding: 20px;
-  width: 100%;
-  background-color: $primary;
-  color: white;
-
-  height: $footer-height;
-  padding: 20px;
-  @include is-mobile {
-    height: $footer-height-mobile;
-  }
+/*         Home Footer                */
+$scale-hover:scale(1.2);
+
+$breakpoint-md:1200px;
+$breakpoint-sm:700px;
+$fontsize-paragraph:1.2rem;
+$clippath-botRight-leftTop:polygon(14% 0, 100% 0, 100% 85%, 86% 100%, 0 100%, 0 15%);
+$transition-5:all .5s ease-in-out;
+$transition-3:all .3s ease-in-out;
+
+
+.footer-section{
+    background-color: $dark-background;
+    padding-top: 175px!important;
+    .footer-container{
+        width: 80%;
+        margin: auto;
+    }
+}
+
+.footer-leftside{
+    padding-bottom: 20px;
+        p{
+            color: $light-gray;
+            font-size: $fontsize-paragraph;
+            font-family: Segoe UI !important;
+            @include is-desktop {
+                font-size: 1rem;
+            }
+            @include is-mobile{
+                font-size: .8rem;
+            }
+        }
+        i{
+            color: $header-item;
+            padding-right: 5px;
+        }
+        @include is-mobile {
+            max-width: 100%!important;
+            flex: 100%;
+            text-align: center;
+        }
+        @include is-tablet {
+            max-width: 100%!important;
+            flex: 100%;
+            text-align: center;
+        }
+}
+
+
+.footer-socialmedia{
+    padding-bottom: 100px;
+
+
+    .footer-followus{
+        width: 108px!important;
+        position: relative;
+        color: $header-item;
+        padding-bottom: 25px;
+        font-family: Segoe UI !important;
+        font-weight: bolder;
+        font-size: 20px;
+            &::after{
+              content: "";
+              position: absolute;
+              bottom: 0;
+              top: 28px;
+              left: 0px!important;
+              height: 4px;
+              width: 56%!important;
+              background: $underline-cyan;
+              transition: $transition-5;
+            }
+        @include is-mobile {
+            margin: auto!important;
+        }
+        @include is-tablet {
+            margin: auto!important;
+        }
+        @include is-desktop {
+            margin: auto!important;
+        }
+    }
+    
+    .socialMedia{
```

Comment:
just use `justify-content: center;` if it used in all mixins with the same value and remove all mixins

Reply:
its not in the center in the big screens only in the desktop and below

Reply:
put it outside and change `justify-content` NOT to be `center` in the big screens